### PR TITLE
Added URL to media detail view

### PR DESCRIPTION
**Pull request by mjduijn**
Added a clickable, copyable URL to the media library detail view

Fixes wordpress-mobile/WordPress-Android#2069

### CODE REVIEW

**Comment by mjduijn** [comment]
Hey everyone, this is my first pull request and I was wondering if everything was ok.
As far as I know I've followed your standards but please let me know if something is wrong.

File: WordPress/src/main/java/org/wordpress/android/ui/media/MediaItemFragment.java [hunk 1]
```diff
@@ -203,8 +205,13 @@ private void refreshViews(Cursor cursor) {
             String fileType = fileURL.replaceAll(".*\\.(\\w+)$", "$1").toUpperCase();
             mFileTypeView.setText("File type: " + fileType);
             mFileTypeView.setVisibility(View.VISIBLE);
+
+            // set the file URL
+            mFileUrlView.setText("File URL: " + fileURL);
```
Comment (maxme) [inline WordPress/src/main/java/org/wordpress/android/ui/media/MediaItemFragment.java]: `File URL: ` must be declared in `strings.xml` for i18n.

File: WordPress/src/main/java/org/wordpress/android/ui/media/MediaItemFragment.java [hunk 2]
```diff
@@ -203,8 +205,13 @@ private void refreshViews(Cursor cursor) {
             String fileType = fileURL.replaceAll(".*\\.(\\w+)$", "$1").toUpperCase();
             mFileTypeView.setText("File type: " + fileType);
```
Comment (maxme) [inline WordPress/src/main/java/org/wordpress/android/ui/media/MediaItemFragment.java]: Same here. That view is probably super old.

**Comment by maxme** [comment]
Pull request looks good, but we have to replace all hardcoded text in `MediaItemFragment` and move them to `strings.xml`

**Comment by maxme** [comment]
Strings to move to `strings.xml`:
```
"Added on: "
"Uploaded on: "
"File name: "
"File type: "
"File URL: "
"Dimensions: "
```

**Comment by mjduijn** [comment]
Thanks for the feedback, I've replaced all hardcoded strings in the view and moved them to strings.xml.
The values are quoted to keep the trailing space but if this causes a problem with translations, let me know

**Comment by maxme** [comment]
Yes please, remove the quotes and concatenate the space in code. I fear translators will be confused by the quotes.

For example:
`<string name="media_details_added_on">Added on:</string>`
and 
`mDateView.setText(getResources().getString(R.string.media_details_added_on) + " " + date);`

**Comment by maxme** [comment]
Thanks!
:shipit:

**Pull request merged**